layouts/shortcodes/newsletter/papost-form.html

```diff
@@ -10,52 +10,60 @@
     data-ga-category="newsletters:papost-full"
   >
     {{ partial "newsletter/papost-hidden.html" . }}
-    <h2 class="title is-3">Sign Up for PA Post</h2>
+    <h2 class="title is-3 font-sans text-3xl mt-7 mb-6 font-bold">
+      Sign Up for PA Post
+    </h2>
     <div class="field">
-      <label class="label" for="{{ $idFname }}">First Name</label>
+      <label class="label font-sans font-black" for="{{ $idFname }}"
+        >First Name</label
+      >
     </div>
     <div class="field">
       <p class="control is-expanded">
         <input
           id="{{ $idFname }}"
-          class="input"
+          class="input border font-sans w-full border-s-4 rounded p-1"
           type="text"
           name="FNAME"
           required
         />
       </p>
     </div>
     <div class="field">
-      <label class="label" for="{{ $idLname }}">Last Name</label>
+      <label class="label font-sans font-black" for="{{ $idLname }}"
+        >Last Name</label
+      >
     </div>
     <div class="field">
       <p class="control is-expanded">
         <input
           id="{{ $idLname }}"
-          class="input"
+          class="input border font-sans w-full border-s-4 rounded p-1"
           type="text"
           name="LNAME"
           required
         />
       </p>
     </div>
-    <div class="field">
+    <div class="field mb-3">
       <label class="label" for="{{ $idEmail }}">Email Address</label>
```

Comment:
This one was missed. 

![image](https://user-images.githubusercontent.com/222245/210648043-ba34a2e6-9c28-4d3b-8caa-1f642a0d3006.png)